### DISCUSSION TITLE
Support of the GC9107 driver IC?

### DISCUSSION THREAD

**Discussion by michcfr** [moononournation/Arduino_GFX#179] [General]
Hello, 
The GC9107 driver IC is not included in the supported display list. My display is: 128x128 - 262K color
May we reuse the already existing code for the drivers: GC9A01, GC9106, GC9503V?
If not, does somebody plan to support it?

regards,
Michel

**Comment by moononournation** [comment]
I did not have GC9107 in hand

**Comment by michcfr** [comment]
thank you for your response.
This ESP32 board has the GC9107 driver: https://github.com/Xinyuan-LilyGO/T-QT
Maybe you can reuse code?

**Reply by moononournation** [reply]
please get update and test it
